tests/NUnit.OneTimeSetup.DreddLogs.Tests/Tests.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.Xml;
+using FluentAssertions;
+using FluentAssertions.Execution;
+using NUnit.Engine;
+using NUnit.Framework;
+
+namespace NUnit.OneTimeSetup.DreddLogs.Tests
+{
+    [TestFixture]
+    public class Tests
+    {
+        private ITestRunner _testRunner;
+
+        [OneTimeSetUp]
+        public void OneTimeSetup()
+        {
+            var testPackage = new TestPackage($"NUnit.OneTimeSetup.DreddLogs.Tests.Internal.dll");
+            _testRunner = TestEngineActivator.CreateInstance().GetRunner(testPackage);
+        }
+
+        [Test]
+        public void SimpleTest()
+        {
+            var testCaseCount = _testRunner.CountTestCases(TestFilter.Empty);
+
+            var xmlReport = _testRunner.Run(null, TestFilter.Empty);
+            var testCaseNodes = xmlReport.SelectNodes("//test-case");
+
+            using (new AssertionScope())
+            {
+                testCaseNodes.Count.Should().Be(testCaseCount);
+
+                foreach (var testCaseNode in testCaseNodes)
+                {
+                    var xmlNode = testCaseNode as XmlNode;
+
+                    var isFailed = xmlNode.SelectSingleNode("./failure") != null;
```

Comment:
How about "is not null" once you use .net5?;)

Reply:
I knew you would notice that :)
Will change it